slotting/src/Cardano/Slotting/SlotBounded.hs

```diff
@@ -61,6 +66,23 @@ data SlotBounded (bounds :: Bounds) a
       }
   deriving (Eq, Functor, Show, Generic, Serialise, NoUnexpectedThunks)
 
+instance (FromCBOR a, Typeable b) => FromCBOR (SlotBounded b a) where
+  fromCBOR = do
+    CBOR.decodeListLenOf 3
+    SlotBounded
+      <$> fromCBOR
+      <*> fromCBOR
+      <*> fromCBOR
+
+instance (ToCBOR a, Typeable b) => ToCBOR (SlotBounded b a) where
+  toCBOR SlotBounded {sbLower, sbUpper, sbContent} =
+    mconcat
+      [ CBOR.encodeListLen 3,
+        toCBOR sbLower,
+        toCBOR sbUpper,
+        toCBOR sbContent
+      ]
```

Comment:
I don't know what the convention is in this repo, but in ouroboros-consensus, we start lines with commas.

Reply:
:heavy_plus_sign: